infra/aws/terraform/prow-build-cluster/resources/kube-system/tune-sysctls_daemonset.yaml

```diff
@@ -0,0 +1,47 @@
+# a simple daemonset to tune sysctls
+# intended to be used in a prow build cluster
+# see https://github.com/kubernetes/test-infra/pull/13515 for more info
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: tune-sysctls
+  namespace: kube-system
+  labels:
+    app: tune-sysctls
+spec:
+  selector:
+    matchLabels:
+      name: tune-sysctls
+  template:
+    metadata:
+      labels:
+        name: tune-sysctls
+    spec:
+      hostNetwork: true
+      hostPID: true
+      hostIPC: true
+      tolerations:
+      - operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: setsysctls
+        command:
+        - sh
+        - -c
+        - |
+          while true; do
+            sysctl -w fs.inotify.max_user_watches=524288
+            sleep 10
```

Comment:
This is already done by https://github.com/kubernetes/k8s.io/blob/main/infra/aws/terraform/prow-build-cluster/eks.tf#L105-L107. what's the difference ?

Reply:
I missed that. Copied over the prow-build-cluster contents. Deleting